reporter-web-app/src/components/PackageFindingsTable.js

```diff
@@ -111,12 +100,13 @@ class PackageFindingsTable extends React.Component {
                 defaultSortOrder: 'ascend',
                 key: 'path',
                 sorter: (a, b) => {
-                    const idA = a.path.length;
-                    const idB = b.path.length;
-                    if (idA < idB) {
+                    const lenA = a.path.length;
```

Comment:
I guess this could be just `sorter: (a, b) => { a.path.length - b.path.length }`?

Reply:
Fixed, it can just be `sorter: (a, b) => a.path.length - b.path.length`